package.json

```diff
@@ -122,9 +122,8 @@
     "react-dom": "17.0.1",
     "react-router-dom": "6.11.1",
     "react-string-replace": "1.1.0",
-    "react-virtualized-auto-sizer": "^1.0.6",
-    "react-window": "1.8.7",
-    "react-window-infinite-loader": "^1.0.8"
+    "react-virtuoso": "^4.3.8",
+    "react-window": "^1.8.9"
```

Comment:
This is still used by https://github.com/evergreen-ci/spruce/blob/ebf3b121f76db2f33073a851607d4168db120d92/src/components/VersionRestartModal/TaskStatusCheckboxContainer.tsx#L1  I've opened EVG-20126 to take care of it